FindYourRecipe/FindYourRecipe.Contracts/ICategoryService.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using System;
-using FindYourRecipe.Application.Models.Request;
-using FindYourRecipe.Application.Models.Response;
-using FindYourRecipe.DataAccess;
+using FindYourRecipe.Contracts.Models;
 
-namespace FindYourRecipe.Application.Interfaces
+namespace FindYourRecipe.Contracts
 {
 	public interface ICategoryService
 	{
-		Task<CategoryResponseModel> GetByIdAsync(int id);
+        Task<List<CategoryResponseModel>> GetAsync();
```

Comment:
The way you named your methods is not wrong but, in my opinion, it would be easier to read/maintain if the methods would contain the name of the entity it works with.
Example: 
- `GetCategoryAsync` intead of `GetAsync`
- `DeleteIngredientAsync` instead of `DeleteAsync` and so on